frontend/src/APIClients/lessonAPIClient.ts

```diff
@@ -0,0 +1,61 @@
+import baseAPIClient from "./BaseAPIClient";
+
+export type LessonRequest = {
+  course: string;
+  title: string;
+  description: string;
+  image: string;
+  content: [Record<string, unknown>];
+};
+
+export type LessonResponse = {
+  id: string | number;
+  course: string;
+  title: string;
+  description: string;
+  image: string;
+  content: [Record<string, unknown>];
+};
+
+// TO DO: error handling
+
+const get = async (id: string): Promise<LessonResponse> => {
+  try {
```

Comment:
I'm pretty sure we need to include the bearer token here like we do with EntityAPIClient, that token will be needed for any authenticated routes which the lessons route should be?